docs/source/plugin_quickstart.rst

```diff
@@ -1,81 +1,56 @@
 Quickstart
 ===========
-
 ----------------------------------------------------------------------------------------------------
 Connect the Open edX Commerce plugin for WordPress to the Enrollment API of your Open edX Platform
 ----------------------------------------------------------------------------------------------------
-
 Let us start installing and configuring the Open edx Commerce plugin to connect your WordPress site with the enrollment APIs from your Open edX platform.
-
 Index
 ------
 - `Installation`_
 - `Configure the Plugin`_
-- `Create an Enrollment Request`_
-- `Next Steps`_
+- `Create a Product`_
+- `Create payment method`_
+- `Buy product on LMS as student`_
 
 Installation
 -------------
 
 Requirements
 ^^^^^^^^^^^^^
-
-- PHP 8.0 or greater is recommended
-- Wordpress 6.3.1 or greater
-- `WooCommerce plugin. <https://wordpress.org/plugins/woocommerce>`_
+- Setup https://github.com/edly-io/tutor-contrib-wordpress.git plugin
```

Comment:
This is not a public repository (or it has been deleted since).

Also hosting wordpress via tutor is not a requirement. For the `openedx-wordpress-ecommerce` the only requirements are as listed before.